### ISSUE TITLE
Bug: Unable to drag application windows using Alt

### ISSUE THREAD

**Issue by ajstrongdev** [RamonUnch/AltSnap#581]
OS: Windows 11 Home
Prerequisites: Ensure application shortcut is placed in `C:\Users\Username\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup`

The application should, and does launch during startup (using the startup directory provided), however I am unable to drag/move application windows using Alt unless I manually force restart the application

I can verify that the application successfully launches on system startup, by checking the system tray.

Great project, keep up the great work.

**Comment by RamonUnch** [comment]
Did you try to use the Autostart option in the General tab?

**Comment by ajstrongdev** [comment]
> Did you try to use the Autostart option in the General tab?

No, however I've used this method previously and it hasn't been an issue until I've installed AltSnap on a new device, leading me to believe this behaviour probably comes from a new update.

Will try this and let you know when I next reboot 👍
